app/views/content/shared/teaching/_funding.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="funding">
+  <h3>Fund your teacher training</h3>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_chemistry %> or scholarships of <%= v :scholarships_chemistry %></a> are available for eligible trainee <%= @front_matter["subject"] %> teachers.</p>
```

Comment:
This partial with bursary and scholarship variables isn't going to work on other pages - we'll always be displaying the figures for chemistry if we reference `bursaries_postgraduate_chemistry` and `scholarships_chemistry` here!

Can we add the subject from the front matter into these variables or is that too complex @sarahcrack ?